content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -9,6 +9,32 @@ Now we will set up the buildpack scaffolding.
 
 Let's create the directory where your buildpack will live:
 
+## Using the Pack CLI
+
+The `buildpack new <id>` commmand will create a directory named for the buildpack ID.
+
+Example:
+<!-- test:exec -->
+```bash
+pack buildpack new examples/ruby \
+    --api 0.5 \
+    --name "Ruby Buildpack" \
+    --path ruby-buildpack \
+    --version 0.0.1
+```
+This command will create `ruby-buildpack` directory which contains `buildpack.toml`, `bin/build`,  `bin/detect` files.
+
+### Additional Parameters
+- `-a, --api` Buildpack API compatibility of the generated buildpack
+- `-h, --help` Help for 'new'
+- `--path` the location on the filesystem to generate the artifacts.
+- `--stacks` Stack(s) this buildpack will be compatible with. Repeat for each stack in order, or supply once by comma-separated list
+- `-V, --version` the version of the buildpack in buildpack.toml
+
+
+
+
+## Using Shell
 <!-- test:exec -->
 ```bash
 mkdir ruby-buildpack
```

Comment:
Now that you've added the `pack buildpack new`, we'll also need to change some of the subsequent steps. You'll no longer need to `mkdir` or "create" files. I think this will move be a change to the prose.

Reply:
Okey, i deleted some subsequent steps, and updated generated content with running command